.github/ISSUE_TEMPLATE/testplan.md

```diff
@@ -1161,8 +1161,11 @@ tsh bench web sessions --max=5000 --web user ls
   - [ ] Non-AD instance can be added to `non_ad_hosts` section in config file and is visible in UI
   - [ ] Non-AD can be added as dynamic resource and is visible in UI
   - [ ] Non-AD instance has label `teleport.dev/ad: false`
-  - [ ] Connecting to non-AD instance works with Enterprise license
-  - [ ] Connecting to non-AD instance fails with OSS
+  - [ ] Connecting to non-AD instance works with OSS if there are no more than 5 non-AD desktops
```

Comment:
Sorry for lacking the background on how this feature is offered. This and the below case reads to me that in OSS, we allow for more than 5 non-AD desktops to be added but disallow connecting to them if the added desktop count is more than 5. Is this correct assumption?

Can you also confirm that if there are more than 5 desktops added, we disallow connecting to any one of them? OR we disallow connecting to more than 5 simultaneous desktops?

Reply:
> Can you also confirm that if there are more than 5 desktops added, we disallow connecting to any one of them?

Confirmed.